### DISCUSSION TITLE
Create PRs during traverse?

### DISCUSSION THREAD

**Discussion by tir38** [VirtusLab/git-machete#1240] [Q&A]
IIUC `git machete traverse` will push existing tracked and new untracked branches. If I previously created a PR for a branch via `git machete github create-pr` then that PR will see an update. But if this is a new branch that doesn't have a PR yet, can I get machete to create that PR as part of the traverse process?

**Comment by PawelLipski** [comment]
Makes sense! I'll consider implementing that alongside #1215

**Reply by PawelLipski** [reply]
Opened #1241

**Comment by PawelLipski** [comment]
Update: #1215 is ready (PR #1340). I'll now proceed to #1242, and with these two ready, doing #1240 will be easy

**Comment by PawelLipski** [comment]
In progress: https://github.com/VirtusLab/git-machete/pull/1423

**Reply by PawelLipski** [reply]
Aaand long done at this point
